scripts/publish-mavencentral.gradle

```diff
@@ -146,29 +146,18 @@ publishing {
             }
         }
     }
-    repositories {
-        maven {
-            name = "sonatype"
-
-            def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
-            def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
-            url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
```

Comment:
Is it already configured into the new plugin? Or because we are not supporting Snapshot for now has been removed?

Reply:
I believe the plugin handles this based on the version number, yes. But we're not using snapshots anyway, so it's also fine if it just doesn't handle it 😄